mkosi/util.py

```diff
@@ -139,9 +139,9 @@ def flock(path: Path, flags: int = fcntl.LOCK_EX) -> Iterator[int]:
 
 
 @contextlib.contextmanager
-def flock_or_die(path: Path) -> Iterator[Path]:
+def flock_or_die(path: Path, flags: int = fcntl.LOCK_EX) -> Iterator[Path]:
```

Comment:
The change to this function is only for symmetry with the definition of `flock`, right? Confused me a bit since it's in the same commit.